api-tests/features/p2-strategic-app.feature

```diff
@@ -84,6 +84,7 @@ Feature: M2B Strategic App Journeys
     Then I get an OAuth response
     When I use the OAuth response to get my identity
     Then I get a 'P2' identity
+    And audit events for 'strategic-app-cross-browser-journey' are recorded [local only]
```

Comment:
As I understand it, we should pull this out into its own scenario in audit-events.feature per the new approach (is that right @thebauSoftwire?)